src/js/components/plans/preflightResults.tsx

```diff
@@ -142,10 +144,7 @@ const PreflightResults = ({ preflight }: { preflight: Preflight }) => {
     return null;
   }
 
-  const { failed, message } = getErrorInfo({
-    preflight,
-    label: i18n.t('Pre-install validation'),
```

Comment:
Because this label was getting double-translated, in non-EN languages i18n was looking for a translation key e.g. `Validatie vóór installatie encountered` for the second translation, instead of key `Pre-install validation encountered`.